### PR TITLE
index files with tika

### PR DESCRIPTION
**Pull request by cekk**
@mamico non riesco a metterti tra i reviewer finché @tassoman non ti inserisce o non accetti l'invito ;)

### CODE REVIEW

File: src/rer/solrpush/adapters/file.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class FileExtractor(object):
+    def __init__(self, context):
+        self.context = context
+
+    def get_file_to_index(self):
+        """
+        """
+        if not self.context.file:
+            return ""
+        form = self.context.REQUEST.form
```
Comment (mamico) [inline src/rer/solrpush/adapters/file.py:16]: getting the file from the request ? why ?

Comment (cekk) [reply src/rer/solrpush/adapters/file.py]: I don't remember: probably with previous queue implementation there was a moment where i came here without the file.

I tested it right now and this doesn't happen anymore: cleaned the code in last commit

File: config/test/conf/schema.xml [hunk 2]
```diff
@@ -136,7 +136,7 @@
    <field name="Language" type="string" indexed="true" stored="true" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
    <field name="ModificationDate" type="date" indexed="false" stored="true" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
 <!-- catchall field, containing all other searchable text fields (implemented via copyField further on in this schema -->
-   <field name="SearchableText" type="text_it" indexed="true" stored="false" required="false" multiValued="false" termVectors="false" termPositions="false" termOffsets="false"/>
+   <field name="SearchableText" type="text_it" indexed="true" stored="true" required="false" multiValued="true" termVectors="false" termPositions="false" termOffsets="false"/>
```
Comment (cekk) [inline config/test/conf/schema.xml:139]: @mamico is it ok for you? It seems that tika sets SearchableText as a list of values

Comment (mamico) [reply config/test/conf/schema.xml:139]: multivalued ok. why stored?

Comment (cekk) [reply config/test/conf/schema.xml:139]: only for debug (i copied it from dev configuration)

**Comment by cekk** [comment]
@mamico ping ;)

**Review by mamico — APPROVED** [review]

**Pull request merged**